docs/reference/upgrade/open-ml.asciidoc

```diff
@@ -11,3 +8,6 @@ states:
 POST _ml/set_upgrade_mode?enabled=false
 --------------------------------------------------
 // CONSOLE
+
+If you closed all {ml} jobs before the upgrade, open them and their associated
```

Comment:
Not all jobs necessarily have datafeeds, so I'd suggest changing this to something like:

```suggestion
If you closed all {ml} jobs before the upgrade, open the jobs and start the
```